source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.json

```diff
@@ -17,7 +17,6 @@
         },
         "data": {
           "failCommands": [
-            "isMaster",
```

Comment:
This change needs to be reverted or else this test will fail.

Reply:
The integration tests should all have both `hello` and `isMaster` in the fail commands when needed. Looks like this one got lost during a merge. Fixed.